recipes/lie_learn/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "lie_learn" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AMLab-Amsterdam/lie_learn/archive/v{{ version }}.tar.gz
+  sha256: e7249526cfbe145c3a62477d89c0a8f31af4f3e46b94489456b10a2e59a85398
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython
+    - requests
+    - numpy
+    - scipy
+  run:
+    - python
+    - cython
+    - requests
+    - numpy
+    - scipy
+
+test:
+  requires:
+    - pytest
+  source_files:
+    - lie_learn
```

you are not testing anything here, could you please add
```suggestion
    - lie_learn
  imports:
    - lie_learn
```